setup.cfg

```diff
@@ -36,20 +36,21 @@ python_requires = >= 3.9
 
 [options.extras_require]
 backend-es =
-    elasticsearch==8.4.3
+    elasticsearch>=8.0.0
 backend-ldp =
-    ovh==1.0.0
-    requests==2.28.1
+    ovh>=1.0.0
+    requests>=2.0.0
 backend-mongo =
-    pymongo[srv]==4.3.2
+    pymongo[srv]>=4.0.0
 backend-swift =
-    python-keystoneclient==5.0.1
-    python-swiftclient==4.1.0
+    python-keystoneclient>=5.0.0
+    python-swiftclient>=4.0.0
 backend-ws =
-    websockets==10.3
+    websockets>=10.3
 cli =
-    click==8.1.3
-    click-option-group==0.5.5
+    click>=8.1.0
+    click-option-group>=0.5.0
+    sentry-sdk>=1.9.0
```

Comment:
Are `lrs` dependencies in the scope of this PR?
Could it cause problems having `sentry_sdk==1.10.1` in `cli` and `sentry-sdk>=1.9.0` in `lrs` ?

Reply:
As long as version constrains are not conflicting I would say no, but we need to be careful on this.